sunpy/physics/limb_darkening.py

```diff
@@ -0,0 +1,184 @@
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+
+
+def darklimb_u(wavelength):
+    """
+    Returns the constant, u, for a given wavelength for input into the limb
+    darkening function.
+
+    Parameters
+    ----------
+    wavelength : `~sunpy.map.GenericMap.wavelength`
+        The wavelength for which the limb darkening is to be calculated.
+
+    Notes
+    -----
+    Coefficients for 5th order polynomial fit to limb darkening
+    constant `u` was done on data from Astrophysical
+    Quantities by Allen.
+
+    Limitations
+    -----------
+    Only applies to wavelengths in the range 4000<lambda<15000 Angstrom
+
+    Reference
+    ---------
+    Cox, Arthur N., Allen's astrophysical quantities, 4th ed. Publisher:
+        New York: AIP Press; Springer, 2000. ISBN: 038798746
+
+    """
+
+    au = -8.9829751
+    bu = 0.0069093916
+    cu = -1.8144591e-6
+    du = 2.2540875e-10
+    eu = -1.3389747e-14
+    fu = 3.0453572e-19
+
+    a = np.array((au, bu, cu, du, eu, fu))
```

Comment:
What are these?

Reply:
To model the limb darkening effect, the limb darkening function uses a 5th order polynomial fitting to the limb darkening constants obtained from Astrophysical Quantities.
These are the six constants taken from Allen's book.
For reference : [The IDL algorithm](https://hesperia.gsfc.nasa.gov/ssw/gen/idl/solar/darklimb_correct.prol)